### ISSUE TITLE
First release

### ISSUE THREAD

**Issue by TNO-SlaFleur** [flexiblepower/s2-python#17]
Would be great if we could make a first release with FRBC only. Currently Elaad, Seita and TNO are working with S2 and are focusing on FRBC. A first release would allow all current users/organisation to start integrating this library.

TODO till first release:

- [x] #15 
- [x] #12 
- [x] #11 
- [x] #18 

@victorgarcia98  What are your thoughts here? Too soon or perhaps some issues missing?

**Comment by victorgarcia98** [comment]
That would be awesome! We'd love to integrate it into our [CEM implementation](https://github.com/FlexMeasures/flexmeasures-client/).

I would to the TODO list for the first release:
- [ ] [Add GH action for automated testing + publishing](https://github.com/flexiblepower/s2-ws-json-python/issues/18)

**Comment by TNO-SlaFleur** [comment]
@victorgarcia98  Congratulations, first release has succeeded! https://github.com/flexiblepower/s2-python/actions/runs/6123805600

Ran into an issue where I forgot to change the version number. I corrected by adding tag `v0.0.0` instead of `v0.0.1` so the release on pypi matches the tag in our repo. Added https://github.com/flexiblepower/s2-python/issues/31 to help out preventing this mistake on my part in the future.
